### PR TITLE
Specify production values for doctrine dbal

### PR DESCRIPTION
**Pull request by Nyholm**
Related to #179

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "php": ">=7.4",
         "ext-json": "*",
         "composer/package-versions-deprecated": "^1.11",
-        "doctrine/doctrine-bundle": "2.2.4",
+        "doctrine/doctrine-bundle": "2.3.0",
```
Comment (stof) [inline composer.json:10]: why keeping an exact version now ? Are you still bisecting ?

Comment (Nyholm) [reply composer.json:10]: I just wanted to confirm that this was a fix. I'll follow this up with a second pr to allow the latest versions of all doctrine packages.

**Comment by Nyholm** [comment]
Interesting. This PR did not fix the issue..

**Comment by stof** [comment]
@Nyholm try putting a postgresql string in .env. I think the .env value will still win over this explicit config as DBAL makes the URL win.

**Comment by Nyholm** [comment]
Yepp. I believe you are correct: https://github.com/symfony-tools/carsonbot/pull/192

**Pull request merged**